### PR TITLE
first version of the new print design

### PR DESCRIPTION
**Pull request by jaykayone**

### CODE REVIEW

File: print/print-apps/geoportailv3/config.yaml [hunk 1]
```diff
@@ -1,49 +1,349 @@
 pdfConfig: !pdfConfig
   compressed: false
   author: "Mapfish Print"
-  subject: "Mapfish Print"
+  subject: "Mapfish Print Verbose example"
```
Comment (sbrunner) [inline print/print-apps/geoportailv3/config.yaml]: Not sure that's the wanted subject ...

File: print/print-apps/geoportailv3/config.yaml [hunk 2]
```diff
@@ -1,49 +1,349 @@
...
 
 throwErrorOnExtraParameters: true
 
 templates:
   A4 portrait: !template
     pdfConfig: !pdfConfig
-      title: "A4 Portrait"
-    reportTemplate: report.jrxml
-    tableData: jrDataSource
+      title: "a4_portrait"
```
Comment (sbrunner) [inline print/print-apps/geoportailv3/config.yaml]: Not sure that's the wanted title

Comment (jaykayone) [inline print/print-apps/geoportailv3/config.yaml]: they will be localized, this is a translation key

File: main.mk [hunk 3]
```diff
@@ -1,4 +1,5 @@
 INSTANCE_ID = main 
 VARS_FILE = vars_main.yaml
+PRINT_OUTPUT = /var/lib/tomcat/webapps
```
Comment (jaykayone) [inline main.mk]: the make file now copies the war to the correct directory, but the created file is 700 and tomcat won't unpack it unless i set it to 755 manually...

File: print/print-apps/geoportailv3/config.yaml (hunk 2 above)
Comment (sbrunner) [inline print/print-apps/geoportailv3/config.yaml]: I should miss something because I don't understand how it can work ...

Comment (jaykayone) [inline print/print-apps/geoportailv3/config.yaml]: My understanding (might be wrong though):
The server will send these titles to the client, where they can be translated using ngTranslate.

Comment (jaykayone) [inline print/print-apps/geoportailv3/config.yaml]: OK, I think I understood. What will be translated is the name of the template. This one is going into the PDF.
I'll update them

**Comment by jaykayone** [comment]
@sbrunner Ok to be merged? I think the 700 vs 755 issue should be handeled by you in another PR. If you are OK with this, I will create a separate issue

**Comment by sbrunner** [comment]
For the right issue it will be fix in the next version :-)

**Comment by sbrunner** [comment]
Looks good for me

**Pull request merged**